general/calendars/outlook/outlook_calendar.rst

```diff
@@ -66,8 +66,21 @@ and paste it to *Client Secret* in Odoo.
 Sync with Outlook
 =================
 
+.. important::
+   | Odoo Calendar sends a confirmation email at the creation of an event to its attendees. But, on
+     the **first** sync, Outlook will send confirmation emails of all scheduled events as well
+     (including past ones).
+   | To avoid this situation, we recommend *archiving all past events in Odoo Calendar before the
+     first sync, and removing yourself from future ones* (removing yourself will not create the
+     corresponding event in Outlook). This way, no redundant emails will be sent during
+     the first synchronization.
```

Comment:
I would rephrase this as follows: To avoid this situation, we recommend *archiving all past events and removing yourself from future ones in Odoo Calendar before the first sync*. This will prevent the events from being created in Outlook. This way, no redundant emails will be sent to attendees during the first synchronization.